DeepFried2/layers/BackwardsConvolutionCUDNN.py

```diff
@@ -30,18 +29,20 @@ def __init__(self, nchan_in, nchan_out, filter_size, stride=1, border=0, mode='c
 
         w_shape = (nchan_in, nchan_out) + self.filter_size
         w_fan = (np.prod(self.filter_size)*nchan_out, np.prod(self.filter_size)*nchan_in)
+        w_name = ('Wconv_{},{}@{}' + 'x{}'*(len(w_shape) - 3)).format(*w_shape)
+        self.W = self._addparam(self.w_shape, init, fan=w_fan, name=w_name)
```

Comment:
`self.w_shape` does not exist at this point. `w_shape` should be used here instead.